spec/statesman/machine_spec.rb

```diff
@@ -374,6 +374,38 @@ def after_initialize; end
     end
   end
 
+  describe "#in_state?" do
+    before do
+      machine.class_eval do
+        state :x, initial: true
+        state :y
+        transition from: :x, to: :y
+      end
+    end
+
+    let(:instance) { machine.new(my_model) }
+    subject { instance.in_state?(state) }
+    before { instance.transition_to!(:y) }
+
+    context "when machine is in given state" do
+      let(:state) { "y" }
+
```

Comment:
:scissors: to this whitespace (since you've only got one example in the block, and its a oneliner.